k8s/service_controller/server/nil/storage.go

```diff
@@ -79,6 +79,10 @@ func (s *NilServiceStorage) GetBrokerByService(id string) (*model.ServiceBroker,
 	return nil, nil
 }
 
+func (s *NilServiceStorage) AllServices() ([]*model.Service, error) {
```

Comment:
at some point in the past I was thinking of using it for testing or as a template for other storage implementations. Might be more trouble than it's worth.